content/uiux/concepts/visual-design-elements/visual-design-elements.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: 'Visual Design Elements'
+Description: 'Visual design elements are the building blocks of visual design. Common elements include line, shape, color, texture, space, form, and typography.'
+Subjects:
+  - 'Web Design'
+Tags:
+  - 'UI'
+  - 'UX'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Visual design elements** are the building blocks of [visual design](https://www.codecademy.com/resources/docs/uiux/visual-design). Common elements include line, shape, color, texture, space, form, and typography. Designers decide which elements to include, how to style them, and where to place them on the page.
+
+- **Lines** can be used to divide pages, make shapes, or create textures. By adjusting a line’s properties (such as style, color, and weight) different effects can be achieved.
```

Comment:
Remove blank lines between elements of the bulleted list.

Reply:
Removed